dashboards-observability/public/components/explorer/home_table/history_table.tsx

```diff
@@ -0,0 +1,97 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ *
+ * Modifications Copyright OpenSearch Contributors. See
+ * GitHub history for details.
+ */
+
+import React, { useState, useRef, useEffect } from 'react';
+
+import {
+  EuiBasicTable,
+  EuiSpacer,
+  EuiLink,
+} from '@elastic/eui';
+
+
+interface TableData {
+  savedHistory: any;
```

Comment:
We should add the type for savedHistory instead of "any".  It will help to validate the response from backend.

Reply:
Yeah okay, i'll change that